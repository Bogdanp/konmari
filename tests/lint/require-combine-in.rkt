#lang racket/base

(require (combine-in))

(require racket/string
         (prefix-in combined: (combine-in threading)))

(require (prefix-in combined: (combine-in threading))
         racket/string)

(require (prefix-in combined: (combine-in threading
                                          racket/base))
         racket/string)
