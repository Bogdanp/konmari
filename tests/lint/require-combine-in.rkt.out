/require-combine-in.rkt:3:10:warning:empty combine-in
/require-combine-in.rkt:9:10:warning:require "racket/string" should come before "threading"
/require-combine-in.rkt:12:43:warning:require "racket/base" should come before "threading"
/require-combine-in.rkt:13:10:warning:require "racket/string" should come before "threading"
